lib/ex_sieve/builder/where.ex

```diff
@@ -9,7 +9,7 @@ defmodule ExSieve.Builder.Where do
   def build(query, grouping, binding) do
     exprs = grouping |> List.wrap |> groupings_expr
     :where
-    |> Filter.build(Macro.escape(query), binding, exprs, __ENV__)
+    |> Filter.build(:and, Macro.escape(query), binding, exprs, __ENV__)
```

Comment:
Why do you hardcode this? There is combinator field in grouping something like this:
```
  @spec build(Ecto.Queryable.t, Grouping.t, Macro.t) :: Ecto.Query.t
  def build(query, %Grouping{combinator: combinator} = grouping, binding) when combinator in ~w(and or)a do
    exprs = grouping |> List.wrap |> groupings_expr
    :where
    |> Filter.build(combinator, Macro.escape(query), binding, exprs, __ENV__)
    |> Code.eval_quoted
    |> elem(0)
  end
```
I think, this is more idiomatic way to do that.